include/oneapi/dpl/pstl/hetero/dpcpp/sycl_iterator.h

```diff
@@ -156,14 +158,24 @@ __internal::sycl_iterator<Mode, T, Allocator> begin(sycl::buffer<T, /*dim=*/1, A
 
 template <typename T, typename Allocator, access_mode Mode>
 __internal::sycl_iterator<__internal::_ModeConverter<Mode>::__value, T, Allocator>
-    begin(sycl::buffer<T, /*dim=*/1, Allocator> buf, sycl::mode_tag_t<Mode>, sycl::property::noinit)
+    begin(sycl::buffer<T, /*dim=*/1, Allocator> buf, sycl::mode_tag_t<Mode>,
+#if __SYCL_2020_NOINIT_MACRO
+          sycl::property::no_init)
```

Comment:
I think it needs to fix the Spec according to new changes as well (https://github.com/oneapi-src/oneAPI-spec/blob/main/source/elements/oneDPL/source/pstl.rst#buffer-wrappers).

Reply:
Sure. Just to not forget about it.